src/api/__tests__/File.test.js

```diff
@@ -528,7 +528,11 @@ describe('api/File', () => {
         });
         test('should do nothing if destroyed', () => {
             file.isDestroyed = jest.fn().mockReturnValue(true);
-            file.getFileExtension('id', () => {}, () => {});
+            file.getFileExtension(
+                'id',
```

Comment:
Is the prettier upgrade causing these code style changes?

Reply:
yup